mango-core/src/main/scala/org/bdgenomics/mango/models/TrackedLayout.scala

```diff
@@ -139,15 +191,47 @@ class OrderedTrackedLayout[T: ClassTag](values: Traversable[(ReferenceRegion, T)
   class Track(val initial: (ReferenceRegion, T)) {
 
     val records = new mutable.ListBuffer[(ReferenceRegion, T)]()
-    records += initial
+    val groupPairs = new mutable.ListBuffer[GroupPair]()
+    val idx: Int = numTracks
+
+    if (initial != null)
+      records += initial
+
+    def this() {
```

Comment:
Use case for null constructer is when you are going to go through a list of pairs to add to a track. Because it is not yet known whether the paired reads overlap, there is not a constructer with a list of reads. But this is an option I can look at it today.